scripts/CEE/pcap-collector/README.md

```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
 Parameters:
 - TIME: In seconds. Must be less than or equal to 900 (15 mins)
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: It will add network interface as parameter
 
 ```bash
 # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
 ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
 
+# Capture traffic for a particular interface in a node
+ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=all
```

Comment:
The way to specify all interfaces is `any` in tcpdump (see [link](https://serverfault.com/questions/805006/tcpdump-on-multiple-interfaces)). 

```suggestion
# Capture traffic for all interfaces on a node
ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=any
```